inc/kmer_unordered_set.hpp

```diff
@@ -22,19 +26,30 @@ class KmerSets {
 		// path to kmer parent files as input from command line
 		path hap1_kmer_fa_path;
 		path hap2_kmer_fa_path;
-
-		int hap1_kmer_count;
-		int hap2_kmer_count;
+		float num_hap1_kmers;
+		float num_hap2_kmers;
+		// component and haplotype 
+		int graph_component;
+		int component_haplotype;
+		static const int parent_hap1_int = 0; // hg03 paternal 
+		static const int parent_hap2_int = 1; // hg04 maternal
```

Comment:
the 'int'  type unfortunately doesn't have the same meaning on different machines, so it should usually be replaced with specific size integers (e.g. uint32_t). If you are indexing a container/array, size_t is recommended because it will take on the same limit as the maximum size of the array.

Reply:
Thanks for the info